app/src/sharedTest/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdaptersTest.kt

```diff
@@ -0,0 +1,202 @@
+import org.oppia.android.domain.topic.PrimeTopicAssetsControllerModule
+import org.oppia.android.domain.workmanager.WorkManagerConfigurationModule
+import org.oppia.android.testing.TestImageLoaderModule
+import org.oppia.android.testing.TestLogReportingModule
+import org.oppia.android.testing.junit.InitializeDefaultLocaleRule
+import org.oppia.android.testing.robolectric.RobolectricModule
+import org.oppia.android.testing.threading.TestDispatcherModule
+import org.oppia.android.testing.time.FakeOppiaClockModule
+import org.oppia.android.util.accessibility.AccessibilityTestModule
+import org.oppia.android.util.caching.AssetModule
+import org.oppia.android.util.caching.testing.CachingTestModule
+import org.oppia.android.util.gcsresource.GcsResourceModule
+import org.oppia.android.util.locale.LocaleProdModule
+import org.oppia.android.util.logging.EventLoggingConfigurationModule
+import org.oppia.android.util.logging.LoggerModule
+import org.oppia.android.util.logging.SyncStatusModule
+import org.oppia.android.util.logging.firebase.FirebaseLogUploaderModule
+import org.oppia.android.util.networking.NetworkConnectionDebugUtilModule
+import org.oppia.android.util.networking.NetworkConnectionUtilDebugModule
+import org.oppia.android.util.parser.html.HtmlParserEntityTypeModule
+import org.oppia.android.util.parser.image.ImageParsingModule
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Singleton
+
+/** Test for [AppCompatCheckBoxBindingAdapters] */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(
+  application = AppCompatCheckBoxBindingAdaptersTest.TestApplication::class,
+  qualifiers = "port-xxhdpi"
+)
+
+class AppCompatCheckBoxBindingAdaptersTest {
+
+  @get:Rule
+  val initializeDefaultLocaleRule = InitializeDefaultLocaleRule()
+
+  @get:Rule
+  var activityRule: ActivityScenarioRule<AppCompatCheckBoxBindingAdaptersTestActivity> =
+    ActivityScenarioRule(
+      Intent(
+        ApplicationProvider.getApplicationContext(),
+        AppCompatCheckBoxBindingAdaptersTestActivity::class.java
+      )
+    )
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+    Intents.init()
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  private var colorRgb: Int = Color.valueOf(-0x10000).toArgb()
+
+  @Test
+  fun testSetButtonTint_hasCorrectButtonColor() {
+    activityRule.scenario.onActivity {
+      val appCompatCheckBox: AppCompatCheckBox = getAppCompatCheckBox(it)
+      setButtonTint(appCompatCheckBox, colorRgb)
+      assertThat((appCompatCheckBox.supportButtonTintList)?.defaultColor).isEqualTo((colorRgb))
+    }
+  }
+
+  private fun getAppCompatCheckBox(
+    it: AppCompatCheckBoxBindingAdaptersTestActivity
+  ): AppCompatCheckBox {
+    return it.findViewById(R.id.test_check_box)
+  }
+
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<TestApplication>()
+      .inject(this)
```

Comment:
```suggestion
    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
```

Nit: can this fit on one line?

Reply:
okay